### PR TITLE
Transition from asp.net 5 rc1 to rc2

### PR DESCRIPTION
**Pull request by skofman1**
Changes:
1. Recreated all project files
2. Added to NuGet.config new feeds to support CI build (since rc2 is not out yet, the latest package versions are spread out across myget feeds of different teams).
3.  Remove use of Serilog, since it's not compatible with rc2 yet. Will bring it back once available.
4. Recreated NuGet.Services.Logging as an xproj project, due to https://github.com/NuGet/Home/issues/2754 that blocked pack of csproj with rc2
5. EntityFramework changes: Had to move migrations to NuGet.Services.Staging.Manager from NuGet.Services.Staging.Database since keeping them in Database is no longer supported
6. Startup: added a main method that contains initialization logic
7. Filters: Can't use DI in filters due to this: https://github.com/aspnet/Mvc/issues/4616
8. UTs: work only when ran from command line. Running from VS is blocked due to a bug.

The code was tested for basic create stage => push package => commit stage scenario.

### CODE REVIEW

**Comment by skofman1** [comment]
@maartenba @xavierdecoster @yishaigalatzer

File: src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs [hunk 1]
```diff
@@ -2,36 +2,36 @@
...
 
 using System;
-using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace NuGet.Services.Staging.Manager.Filters
 {
+    public class StageIdFilterServiceAttribute : ServiceFilterAttribute
```
Comment (maartenba) [inline src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs]: Can this be renamed to, e.g. `EnsureStageExistsAttribute` or something? Makes it easier to grasp what it does from reading the controller.

File: src/NuGet.Services.Staging.Manager/Filters/OwnerFilter.cs [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
-using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace NuGet.Services.Staging.Manager.Filters
 {
-    public class OwnerFilter : ActionFilterAttribute
+    public class OwnerFilterServiceAttribute : ServiceFilterAttribute
```
Comment (maartenba) [inline src/NuGet.Services.Staging.Manager/Filters/OwnerFilter.cs]: Can this be renamed to, e.g. `EnsureUserHasAccessToStageAttribute` or something? Makes it easier to grasp what it does from reading the controller.

**Comment by maartenba** [comment]
Looks good as far as I can tell.

File: src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs (hunk 1 above)
Comment (yishaigalatzer) [inline src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs]: Also no point in having service in the name

Comment (yishaigalatzer) [inline src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs]: Also put it in a separate file and flatten the namespaces

File: src/NuGet.Services.Staging.Manager/StageService/IStageService.cs [hunk 3]
```diff
@@ -67,5 +67,11 @@ public interface IStageService
         /// Get progress report for this commit.
         /// </summary>
         BatchPushProgressReport GetCommitProgress(StageCommit commit);
+
+
+        /// <summary>
+        /// Adds a package to the stage
+        /// </summary>
```
Comment (yishaigalatzer) [inline src/NuGet.Services.Staging.Manager/StageService/IStageService.cs]: Don't use namespaces inline

File: src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs (hunk 1 above)
Comment (skofman1) [inline src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs]: Fixed

---
In reply to: [62967208](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967208) [](ancestors = 62967208)

Comment (skofman1) [inline src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs]: Fixed

---
In reply to: [62967521](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967521) [](ancestors = 62967521)

Comment (skofman1) [inline src/NuGet.Services.Staging.Manager/Filters/StageIdFilter.cs]: Fixed

---
In reply to: [62967580](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967580) [](ancestors = 62967580)

File: src/NuGet.Services.Staging.Manager/StageService/IStageService.cs (hunk 3 above)
Comment (skofman1) [inline src/NuGet.Services.Staging.Manager/StageService/IStageService.cs]: fixed

---
In reply to: [62967716](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967716) [](ancestors = 62967716)

File: src/NuGet.Services.Staging.Manager/Filters/OwnerFilter.cs (hunk 2 above)
Comment (skofman1) [inline src/NuGet.Services.Staging.Manager/Filters/OwnerFilter.cs]: fixed

---
In reply to: [62967234](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967234) [](ancestors = 62967234)

**Pull request merged**